ui/components/app/account-menu/index.scss

```diff
@@ -231,7 +231,11 @@
   }
 
   &__name {
-    @include H4;
+    @include H6;
+
+    @include screen-md-min {
+      @include Paragraph;
+    }
```

Comment:
Same for name
14px font-size at action popover view
16px font size at expanded view